### PR TITLE
Officialbindings

### PR DESCRIPTION
**Pull request by Alannables**
what we've been testing'

### CODE REVIEW

File: src/main/java/frc/robot/commands/Arm/GrabPiece.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                  Commands.print("Encoder Pos" + -clawEncoder.getRaw() / 8192. * 360.),
+                  Commands.parallel(
+                      new ClawOpenCloseShuffleBoard(75, 5, clawOpenClose), Commands.waitSeconds(1)),
+                  new ClawEncoderMoveUp(0, clawPivot, clawEncoder, "CUBE"),
+                  new ClawOpenClose(0, 5, clawOpenClose).withTimeout(2)),
+              clawEncoder::posDown));
+    } else {
+      Commands.sequence(
+          Commands.print("START"),
+          // new ClawEncoderMoveDown(-100, clawPivot, clawEncoder, "Cube").withTimeout(1.5),
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/Arm/GrabPiece.java:41]: Good cleanup to do - remove unused code here

File: src/main/java/frc/robot/commands/PlaceCone/AllLime.java [hunk 2]
```diff
@@ -19,7 +19,8 @@ public class AllLime extends SequentialCommandGroup {
   public AllLime(
       DrivetrainSubsystem drivetrainSubsystem,
       PoseEstimatorSubsystem poseEstimatorSubsystem,
-      LimeLightSub limeLightSub) {
+      LimeLightSub limeLightSub,
+      double degrees) {
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/PlaceCone/AllLime.java:23]: What is 'degrees' doing

File: src/main/java/frc/robot/commands/PlaceCone/PlaceHigh.java [hunk 3]
```diff
@@ -42,7 +41,7 @@ public PlaceHigh(
         new StraightWheel(drivetrainSubsystem),
         new PPIDAutoAim(drivetrainSubsystem, limeLightSub, LimeLightConstants.UPPER_DISTANCE_SHOOT),
         new Sideways(drivetrainSubsystem, poseEstimatorSubsystem, limeLightSub),
-        new ClawEncoderMoveDown(-30, clawPivot, clawEncoder, "Cube").withTimeout(.1),
+        new ClawEncoderMoveDown(-80, clawPivot, clawEncoder, "Cube").withTimeout(.1),
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/PlaceCone/PlaceHigh.java:44]: .1 second is quite fast

File: src/main/java/frc/robot/commands/PlaceCone/Rotate.java [hunk 4]
```diff
@@ -114,16 +115,26 @@ public void execute() {
...
     } else {
       angleDone = false;
     }
-    if (Math.abs(limeLight.getTx()) < .3) {
+    if (Math.abs(limeLight.getTx()) < 1) {
       sideDone = true;
     } else {
       sideDone = false;
     }
+    if (sideDone && angleDone) {
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/PlaceCone/Rotate.java:128]: Needs to get fixed, we know it's broken

File: src/main/java/frc/robot/commands/PlaceCone/rotatesideways.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.DrivetrainSubsystem;
+import frc.robot.subsystems.LimeVision.LimeLightSub;
+import frc.robot.subsystems.PoseEstimatorSubsystem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class rotatesideways extends SequentialCommandGroup {
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/commands/PlaceCone/rotatesideways.java:15]: Needs more testing

File: src/main/java/frc/robot/RobotContainer.java [hunk 6]
```diff
@@ -327,17 +332,60 @@ private void configureButtonBindings() {
     // new PathConstraints(2, 2), finalNode, obstacles, AStarMap));
 
     // controller.y().onTrue(straightWheel1);
+    // controllertwo
+    // // Place high //5 , 2.5, 5
+    // .y()
+    // .onTrue(
+    // Commands.sequence(
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/RobotContainer.java:339]: more commented out code

File: src/main/java/frc/robot/subsystems/ArmSubsystems/BoreEncoder.java [hunk 7]
```diff
@@ -44,6 +46,11 @@ public boolean posDown() {
     return Math.abs(rawAngle) < 57;
   }
 
+  public boolean posDown2() {
+    double rawAngle = (-m_encoder.getRaw() / 8192. * 360.);
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/ArmSubsystems/BoreEncoder.java:50]: Should be part the subsystem

**Review by RobertTheBlair — APPROVED** [review]
Minor problems, looks good otherwise

**Review by Bennables — APPROVED** [review]

**Pull request merged**